src/makeRobotAccountant.js

```diff
@@ -6,7 +6,20 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return (value1) => {
+    return (value2) => {
+      if (counter === 3) {
+        counter--;
+
+        return 'Bzzz... Error!';
+      }
```

Comment:
create separate variable for the break condition, now it is unclear what you are doing here